bash/pkg.yaml

```diff
@@ -1,15 +1,87 @@
 name: bash
 dependencies:
   - stage: base
+  - stage: patch
+    runtime: true
 steps:
   - sources:
       - url: https://ftp.gnu.org/gnu/bash/bash-{{ .bash_version }}.tar.gz
         destination: bash.tar.gz
         sha256: "{{ .bash_sha256 }}"
         sha512: "{{ .bash_sha512 }}"
+      # ref https://git.alpinelinux.org/aports/tree/main/bash/APKBUILD?h=3.16-stable
+      - url: https://ftp.gnu.org/gnu/bash/bash-5.1-patches/bash51-001
```

Comment:
just a small set of patches :eyeglasses:

Reply:
yes, bash doesn't release newer versions, instead patches, so this is something that even if renovate updates, there could be patches released which we'll have to pick upstream.